src/content/en/2019/fonts.md

```diff
@@ -0,0 +1,118 @@
+
+ **06.15: Preconnect**
+
+Wow—less than 2% of pages are using preconnect! Given that Google Fonts is at 75%, this should be higher! Developers: if you use Google Fonts, use preconnect!! Google Fonts, proselytize preconnect more!
+
+**06.16: local()**
+
+See `06.14`, my comments are similar. It has been said (and I agree) that using `local()` can be unpredictable as installed versions of fonts can be outdated and unreliable.
+
+**06.18, 06.21, 06.25: Variable Fonts**
+
+Even at 1.8% this was higher than expected—although I’m excited to see this take off even higher. I would be interested in some deeper analysis here. Is a big player using variable fonts? Can we slice this by host instead of requests?
+
+**06.28 and 06.28b: Variable Font variation axes**
+
+Through the lens of the this large data set, these are very low sample sizes—take these results with a grain of salt. However, `opsz` as the most common axis is notable, with `wght` and `wdth` trailing. In my experience, the introductory demos for variable fonts are usually `font-weight` based.
+
+**06.29: Variable Font variation axes +/- 20pt**
+
+- [ ]  [CONTENT_NEEDED]
+
+**06.30: Variable Font variation axes combinations**
+
+- [ ]  [CONTENT_NEEDED]
+
+**06.26: font-stretch**
+
+`font-stretch` is not a well known `@font-face` descriptor, historically, due to poor browser support. That’s changed and it is now well supported. It has been suggested that using condensed fonts on smaller viewports allows more text to be viewable, but it certainly isn’t common. That being said, that this descriptor is used more on desktop than mobile is unexpected—and ~7% seems much higher than I would have predicted.
+
+**06.31: Top Fonts**
+
+It’s super weird that the top fonts here seem to match up very similarly to the list on [https://fonts.google.com/?sort=popularity](https://fonts.google.com/?sort=popularity)
+
+**06.35: Page with the most font requests**
+
+After diving into the code, the site that requests 718 web fonts is actually requesting 718 Google Fonts! It looks like a malfunctioning “Above the Page fold” optimization plugin for WordPress has gone rogue on this site and is requesting all the Google Fonts—oops!
+
+**06.42 and 06.43: color fonts**
+
+Usage here of these is basically nonexistent but you can check out the excellent resource [Color Fonts! WTF?](https://www.colorfonts.wtf/) for more information. Similar (but not at all) to the SVG format for fonts (which is bad and going away), this allows you to embed SVG inside of OpenType files (which is awesome and cool).
+
+**06.46: Uses a Google Fonts link**
+
+- [ ]  *Zach’s note: If the intention here was to measure if pages used an optimized single CSS request for the Google Fonts stylesheet (versus a bunch of disparate ones)—was that what was measured?*
+- [ ]  [CONTENT_NEEDED]
+
+**06.47: Uses a Google Fonts link first**
+
+- [ ]  [CONTENT_NEEDED]
```

Comment:
The Google Fonts "Getting Started" page recommends placing a HTML `<link>` element to the Google Fonts API as the very first element within `<head>` to optimize latency, but many people do not know about or do this.